src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -111,20 +112,21 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
     refreshPanelsVisibility()
 
     addWelcomeMessage()
-    refreshSubscriptionTab()
+    ApplicationManager.getApplication().executeOnPooledThread { refreshSubscriptionTab() }
     loadNewChatId()
   }
 
+  @RequiresBackgroundThread
   fun refreshSubscriptionTab() {
-    runInEdtAsync(Disposer.newCheckedDisposable()) {
```

Comment:
tbh, I think we shouldn't have it called on Edt (whatever means "Async" here). for an experiment I switched `Thread.sleep(200)` to wait a few seconds instead - I was freezing the UI 😕 

I marked this method as `RequiresBackgroundThread` and called it on     `ApplicationManager.getApplication().executeOnPooledThread` where we call it.